tensorboard/components/tf_backend/router.ts

```diff
@@ -22,7 +22,7 @@ namespace tf_backend {
       params?: URLSearchParams
     ) => string;
     pluginsListing: () => string;
-    runs: () => string;
+    runs: (experiment?: string) => string;
     runsForExperiment: (id: tf_backend.ExperimentId) => string;
```

Comment:
This machinery is unrelated; I’ll remove it in a future change.

I didn’t want to reuse it because it assumes some things about both
inputs (e.g., `type ExperimentId = number`) and outputs (the output of
the `/experiment_runs` route is not just a list of string names) that
may not be what we want.